docs/src/examples/components/Menu/Usage/MenuExampleKind.shorthand.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { Menu, MenuItemKindOptions } from '@stardust-ui/react'
+
+const items = [
+  { key: 'editorials', content: 'Editorials' },
+  { key: 'review', content: 'Reviews' },
+  { key: 'divider', kind: 'divider' as MenuItemKindOptions },
+  { key: 'events', content: 'Upcoming Events' },
+]
+
+const MenuExampleKind = () => (
+  <Menu defaultActiveIndex={0} items={items} vertical pointing="start" />
+)
+
+export default MenuExampleKind
```

Comment:
```suggestion
export default MenuExampleDivider
```

As I understand this example should show us the usage of `MenuDivider`, SUI uses `Content` section of docs to show elements usage: https://semantic-ui.com/collections/menu.html#header